doc-content/enterprise-only/getting-started/business-process-proc.adoc

```diff
@@ -26,77 +26,95 @@ The *Package* specifies the location inside the existing project where the asset
 . Enter the following values:
 +
-* *Name*: `application`.
+* *Name*: `application`
 * *Data Type*: `Application [com.myspace.mortgage_app]`
 
 == Creating outgoing connections and exclusive gateways
-Create an outgoing connection to an exclusive gateway. Exclusive gateways are used to make decisions and react to events based on the available data.
+This section shows you how to create outgoing connections, exclusive gateways, and business rule tasks. Exclusive gateways are used to make decisions and react to events based on the available data.
 
 .Procedure
-. Click on the start event and create an outgoing connection from the start event to an exclusive gateway.
+
+. Create an outgoing connection from the start event to an exclusive gateway:
+.. Click the start event and click the *Create Parallel* icon.
++
+image::getting-started/create-parallel.png[]
+.. Hover over the image:getting-started/config-icon.png[] icon of the parallel and click the *Convert into Exclusive* icon.
 +
 image::getting-started/exclusive-gateway.png[]
 
-. Create an outgoing connection from the exclusive gateway to a task.
+. Create an outgoing connection from the exclusive gateway to a business rule task:
+.. Click the exclusive gateway and click the *Create Task* icon.
 +
 image::getting-started/create-task.png[]
-
-. Convert the new task to a business rule task.
+.. Hover over the image:getting-started/config-icon.png[] icon of the task and click the *Convert into Business Rule* icon.
 +
 image::getting-started/bus-rule-task.png[]
 
-. Click on the business rules task and enter `Validation` in the *Name* field of the *Diagram properties* panel.
-. Expand *Implementation/Execution*, select *validation* in the *Rule Flow Group* field.
-. Define the following Java expression in the *On Exit Action* field :
+. Configure the business rule task:
+
+.. Click the business rules task.
+.. If the *Diagram properties* panel is not open, in the upper-right corner click the *Diagram properties* image:getting-started/diagram_properties.png[] icon.
+.. In the *Diagram properties* panel, enter `Validation` in the *Name* field.
+.. Expand *Implementation/Execution* and from the *Rule Flow Group* list select *validation*.
+.. In the *On Exit Action* field, enter the following Java expression:
 +
 [source,java]
 ----
 System.out.println(application.getProperty());
 ----
-+
-. Scroll down and expand the *Data Assignments* section and click image:getting-started/btn_assign.png[] next to *Assignments*.
-. In the *Validation Data I/O* window, click *Add* to create the following assignments:
+.. Expand *Data Assignments* and click image:getting-started/btn_assign.png[] next to *Assignments*.
+.. In the *Validation Data I/O* window, click *Add* and create the following assignments:
 +
 image::getting-started/val-data-io.png[]
 
-. Click *Save*.
-. In the diagram editor, click *Save*, and *Save*, to confirm your changes.
+.. Click *Save*.
```

Comment:
So click save 3 times?

Reply:
haha, yeah, first saves the configuration and returns, second is to save the project, and third save confirms.

Reply:
Sigh! Ok. Can you specify that the second save is for the project?

Reply:
Done